nix/packages/moq.nix

```diff
@@ -0,0 +1,32 @@
+{ self, nixpkgs, flake-utils, ... }:
+flake-utils.lib.eachDefaultSystem (system:
+  let
+    pkgs = import nixpkgs {
+      inherit system;
+    };
+  in
+    with pkgs;
+    {
+      packages = rec {
+        moq-relay = rustPlatform.buildRustPackage rec {
+          pname = "moq-relay";
+          version = "0.6.7";
+
+          src = ../../.;
+
+          cargoHash = "sha256-kOAulF1OqR1VBKSy15RURJEk2ZpgZIFxPwrr03RbvPk=";
+
+          nativeBuildInputs = [ pkg-config ];
+
+          buildInputs = [ libressl ];
```

Comment:
I'd be curious to know why libressl instead of openssl?

Reply:
libressl is a clean re-write of openssl with supposedly fewer bugs. The API is equivalent to openssl so it works the same. It is also the default ssl impelementation used in upstream nixpkgs